app.js

```diff
@@ -10,6 +10,11 @@
 var express = require('express');
 var db = require('./db');
 var app = express();
+var adapterManager = require('./adapter-manager.js');
+var bodyParser = require('body-parser');
```

Comment:
It turns out that you need a bodyParser of some kind in versions of node > 4. They're no longer included by default.